vortex-duckdb/src/exporter/fixed_size_list.rs

```diff
@@ -32,7 +32,13 @@ pub(crate) fn new_exporter(
     array: &FixedSizeListArray,
     cache: &ConversionCache,
 ) -> VortexResult<Box<dyn ColumnExporter>> {
-    let elements_exporter = new_array_exporter(array.elements(), cache)?;
+    let elements_exporter = new_array_exporter_with_flatten(array.elements(), cache, true)?;
```

Comment:
Same here, voting to have one explicit exporter new. Could consider an enum instead of bool to make this more readable at the call site.